shm/coordinates.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <string>
+
+class Coordinates {
+   public:
+    Coordinates(int positionX, int positionY);
+    bool operator==(Coordinates& position);
+
```

Comment:
There could be a getter and setter for positonX_ and positionY_ ?

Reply:
No, as coordinates are always a pair of x,y. so set/get coordinates only makes sense

Reply:
I am not sure if I understood you correctly. I created getter in Coordinates class to get positionX_ and positionY_ since they are class parameters. If you have any suggestion please tell :)

Reply:
Ok, you are right, I wasn't sure.